vscode-debugger/src/client.ts

```diff
@@ -0,0 +1,23 @@
+import { Client } from '@temporalio/client';
+import { nanoid } from 'nanoid';
+import wait from 'waait';
```

Comment:
If we don't care about node 14 you can use
```
import { setTimeout } from 'node:timers/promises';
```

Reply:
i like keeping 14 support in samples as long as SDK supports it